es-ds-components/components/es-modal.vue

```diff
@@ -10,6 +10,10 @@ const props = defineProps({
         type: Boolean,
         default: false,
     },
+    hideHeader: {
+        type: Boolean,
+        default: false,
+    },
```

Comment:
Rather than adding a new prop for this, it might be simpler to check whether they've passed in that `<template #modal-title> Arcadia Reviews </template>` slot or not.

You can do `const slots = useSlots();` in here to get access to the slots, and then doing a boolean check on `slots['modal-title']` should tell you whether or not the slot was provided.